README.md

```diff
@@ -1,3 +1,99 @@
-# Node.js Express Web Service using standard `node` image
+# Hardening Docker Workloads using Slim CLI
 
-TO-DO
+## The Problem and the Magic 🪄
+Hello folks and if you are reading this, you might have already flashed a gaze at the picture below but before we come to that, let's take a look at the repository and it basically consists of just a simple (but huge) node app and a Dockerfile majorly. Now let's come back to the image (pun intended ofcourse) below again. The left side is what how the original image of our node app looks like and as visible, its huge 995 MB. Now what if we say that you can get the same app's hardened image with size minified to just 89 MB and vulnerabilites reduced by 99.67% and that too in just few simple steps? Sounds magical right? Well, so says the image on your right and that's the magic we are going to talk about today! 
+
+![image](https://user-images.githubusercontent.com/45476902/218093055-50a44810-db1a-43fd-a71d-909e521d4a55.png)
+
+
+### Prerequisites
+* A fresh version of the Slim CLI is installed and configured as described on this [portal](https://portal.slim.dev/cli)
```

Comment:
`nstalled and configured` can be just a link on itself (and then trim the rest of the sentence).